reference/platform-app-yaml.md

```diff
@@ -174,7 +174,7 @@ It has a few subkeys which are:
 
 #### Locations
 
-The `locations` key allows you to provide specific parameters for different URL prefixes.
+The `locations` key allows you to provide specific parameters for different URL prefixes. Each entry's key is an absolute URL path, and its value is configuration directives for that path.  That is, if your domain is `example.com` then `"/"` means "requests for `example.com/`", while `"/admin"` means "requests for `example.com/admin`".
```

Comment:
Perhaps ... "relative URL with a leading slash"? I'm not sure what is absolute URL path ...